system/build_antic.jl

```diff
@@ -0,0 +1,89 @@
+import LoadFlint
```

Comment:
We are not using LoadFlint anymore, though -- or at least won't be once all deps had a release.